content/en/docs/concepts/services-networking/service.md

```diff
@@ -436,8 +436,16 @@ The available `type` values and their behaviors are:
   No proxying of any kind is set up.
 
 The `type` field in the Service API is designed as nested functionality - each level
-adds to the previous.  This is not strictly required on all cloud providers, but
-the Kubernetes API design for Service requires it anyway.
+adds to the previous. For example, in a `NodePort` Service you'll get a `ClusterIP`
+Service by default, because the `NodePort` Service extends the `ClusterIP` Service.
+
+There are a few exceptions for this nested design, such as the `ExternalName` Service.
+The `ExternalName` Service doesn't extend any other type of Service, instead it allows you
+to map an external hostname/IP to an internal DNS record. Another exception is the
+`LoadBalancer` Service, usually the `LoadBalancer` Service extends the `NodePort` Service.
+But you can also define a `LoadBalancer` Service by
+[disabling the load balancer `NodePort` allocation.](https://kubernetes.io/docs/concepts/services-networking/service/#load-balancer-nodeport-allocation)
```

Comment:
```suggestion
[disabling the load balancer `NodePort` allocation.](/docs/concepts/services-networking/service/#load-balancer-nodeport-allocation)
```

Reply:
I don't think this PR is adding much value to this page.
All these information "added" are mentioned later in this page,
or in a related reference page.

Reply:
@tengqm  Any suggestion on how to fix #41173?

Reply:
Just clarify that LoadBalancer Service can be created without having to create
a NodePort one should be okay.

Reply:
> Just clarify that LoadBalancer Service can be created without having to create
> a NodePort one should be okay.

I have simplified it.